internal/driver/driver_test.go

```diff
@@ -63,6 +63,7 @@ func TestParse(t *testing.T) {
 		{"text,lines,cum,show=[12]00", "cpu"},
 		{"text,lines,cum,hide=line[X3]0,focus=[12]00", "cpu"},
 		{"topproto,lines,cum,hide=mangled[X3]0", "cpu"},
+		{"topproto,lines", "cpu"},
```

Comment:
Could you confirm this test fails without your fix? I am curious why the test one line above did not reproduce the issue.

Reply:
I've confirmed this.
I think the line above doesn't reproduce because the hide filter hides all the repeated functions.

Reply:
Interesting that topproto format applied the filters, unlike the proto one. Relevant to the other discussion where we decided to stop discarding the filters for the proto format.